<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndirectSymmetrise</class>
 <widget class="QWidget" name="IndirectSymmetrise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_151">
   <item>
    <widget class="QGroupBox" name="gbInput">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_21">
      <item>
       <widget class="MantidQt::MantidWidgets::DataSelector" name="dsInput" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="showLoad" stdset="0">
         <bool>false</bool>
        </property>
        <property name="workspaceSuffixes" stdset="0">
         <string>_red</string>
        </property>
        <property name="fileBrowserSuffixes" stdset="0">
         <string>_red.nxs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOptions">
     <property name="title">
      <string>Symmetrise</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="properties"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="plotRaw"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbPreview">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <layout class="QVBoxLayout" name="loPreviewProperties">
        <item>
         <widget class="QPushButton" name="pbPreview">
          <property name="text">
           <string>Preview</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="propertiesPreview"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="plotPreview"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOutput">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout1">
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="ckVerbose">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Verbose</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_0">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>165</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="ckPlot">
        <property name="text">
         <string>Plot Result</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>164</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="ckSave">
        <property name="text">
         <string>Save Result</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/DataSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
